packages/ses/src/get-anonymous-intrinsics.js

```diff
@@ -135,6 +118,33 @@ export const getAnonymousIntrinsics = () => {
     '%InertCompartment%': InertCompartment,
   };
 
+  try {
+    // 25.3.1 The AsyncGeneratorFunction Constructor
+
+    // eslint-disable-next-line no-empty-function
+    const AsyncGeneratorFunctionInstance = new FERAL_FUNCTION(
+      'return (async function* AsyncGeneratorFunctionInstance() {})',
+    )();
+    const AsyncGeneratorFunction = getConstructorOf(
+      AsyncGeneratorFunctionInstance,
+    );
+
+    // 25.3.2.2 AsyncGeneratorFunction.prototype
+    const AsyncGenerator = AsyncGeneratorFunction.prototype;
+    // 25.5.1 Properties of the AsyncGenerator Prototype Object
+    const AsyncGeneratorPrototype = AsyncGenerator.prototype;
+    const AsyncIteratorPrototype = getPrototypeOf(AsyncGeneratorPrototype);
+
+    assign(intrinsics, {
+      '%AsyncGenerator%': AsyncGenerator,
+      '%InertAsyncGeneratorFunction%': AsyncGeneratorFunction,
+      '%AsyncGeneratorPrototype%': AsyncGeneratorPrototype,
+      '%AsyncIteratorPrototype%': AsyncIteratorPrototype,
+    });
+  } catch {
+    //
+  }
```

Comment:
add comment why this is here

Reply:
done in https://github.com/endojs/endo/pull/2334/commits/f2bc8f273c1185144710b6c0a6e51a5c94e402b4